sim_pipeline/RSP.py

```diff
@@ -115,14 +119,51 @@ def lens_inejection(lens_pop, ra, dec, num_pix, delta_pix, butler, flux=None):
         mat[:2,:2] = wcs_r.getCdMatrix()
 
         transform = Affine2D(mat)
-        #arr_r = np.copy(image_r.image.array)
+        arr_r = np.copy(image_r.image.array)
 
         _add_fake_sources(image_r, [(point_r, gsobj)])
         inj_arr_r = image_r.image.array
         injected_final_image.append(inj_arr_r)
-        band_report.append(band)
+        #band_report.append(band)
         box_center.append(center)
+        cutout_image.append(arr_r)
+        lens_image.append((inj_arr_r-arr_r))
 
-    t = Table([injected_final_image, band_report, box_center], names=('lens', 'band', 'cutout_center'))
+    t = Table([[lens_image[0]], [cutout_image[0]],[injected_final_image[0]], [injected_final_image[1]], [injected_final_image[2]], [box_center[0]]], names=('lens','cutout_image','injected_lens_r', 'injected_lens_g', 'injected_lens_i', 'cutout_center'))
     return t
+
+def random_ra_dec(ra_min, ra_max, dec_min, dec_max, n):
```

Comment:
please describe what this function does.
Also, this function might be more suited in a utility file instead than in the RSP.py file, since it is rather generic